rkt/image/fetcher.go

```diff
@@ -77,6 +79,13 @@ func (f *Fetcher) fetchImageDeps(hash string) error {
 		}
 		f.addImageDeps(hash, imgsl, seen)
 	}
+
+	if common.SupportsOverlay() {
```

Comment:
This will work because `fetchImageDeps()` is currently always called except when using one of the `--stage1-*` flags.

But yeah, it makes more sense to put it at the end of `FetchImage()`, I thought I did that but apparently I was wrong :flushed:. I'll move that.